instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/AttributeKeys.java

```diff
@@ -26,23 +24,22 @@
             }
           };
 
-  public static final AttributeKey<Context> PARENT_CONNECT_CONTEXT_ATTRIBUTE_KEY =
-      attributeKey("io.opentelemetry.javaagent.instrumentation.netty.v4_1.parent.connect.context");
+  public static final AttributeKey<Context> CONNECT_CONTEXT_ATTRIBUTE_KEY =
+      attributeKey(AttributeKeys.class.getName() + "connect.context");
 
   /**
    * This constant is copied over to
    * io.opentelemetry.javaagent.instrumentation.ratpack.server.TracingHandler, so if this changes,
    * that must also change.
    */
   public static final AttributeKey<Context> SERVER_ATTRIBUTE_KEY =
-      attributeKey(HttpServerTracingHandler.class.getName() + ".context");
+      attributeKey(AttributeKeys.class.getName() + ".context");
 
-  // TODO understand and change to context
   public static final AttributeKey<Span> CLIENT_ATTRIBUTE_KEY =
-      attributeKey(HttpClientTracingHandler.class.getName() + ".span");
```

Comment:
removed `TODO` because I checked and this attribute is being used to pass Span and call end on it, no need to pass Context